packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -64,6 +24,194 @@ export enum AttachState {
     Detached = "Detached"
 }
 
+// @internal
```

Comment:
Do we export @internal interface?

Reply:
As noted in the index file ( https://github.com/microsoft/FluidFramework/pull/20554/files#diff-0cf38bc54b625bd919add7591667aebf30c4e11ab43b20ceed535fe3139203e4R280-R281 ) AttributionInfo is linked in documentation of an exported type, and thus has to be exported. There are two links to it from alpha types in packages/runtime/runtime-definitions/src/attribution.ts .

In my opinion thats a bug in runtime-definitions, and its unclear to me why API extractor doesn't error in runtime-definitions because of it.

Reply:
Fix (to make it not internal) in https://github.com/microsoft/FluidFramework/pull/20811